src/System.ServiceModel.Primitives/ref/System.ServiceModel.Primitives.cs

```diff
@@ -4,7 +4,12 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
-
+namespace System.IdentityModel.Selectors {
+  public abstract partial class X509CertificateValidator {
+    protected X509CertificateValidator() { }
+    public abstract void Validate(System.Security.Cryptography.X509Certificates.X509Certificate2 certificate);
+  }
+}
```

Comment:
Is there any issue introducing this abstract class that is a subset of the one Desktop?  The Desktop version also implements ICustomIdentityConfiguration.